app/javascript/lib/hotwire-livereload-scroll-position.js

```diff
@@ -11,14 +11,17 @@ export function save() {
   localStorage.setItem(KEY, pos.toString())
 }
 
-export function reset() {
-  localStorage.setItem(KEY, "0");
+export function remove() {
+  localStorage.removeItem(KEY, "0");
```

Comment:
No need for value anymore.

```suggestion
  localStorage.removeItem(KEY)
```